src/Microsoft.PowerApps.TestEngine/Modules/TestEngineExtensionChecker.cs

```diff
@@ -216,6 +217,17 @@ private List<TestEngineTrustSource> GetTrustedSources(TestSettingExtensions sett
             return sources;
         }
 
+        /// <summary>
+        /// Validate that the provided provider file is allowed or should be denied based on the test settings
+        /// </summary>
+        /// <param name="settings">The test settings that should be evaluated</param>
+        /// <param name="file">The .Net Assembly file to validate</param>
+        /// <returns><c>True</c> if the assembly meets the test setting requirements, <c>False</c> if not</returns>
+        public virtual bool ValidateProvider(TestSettingExtensions settings, string file)
+        {
+            var contents = GetExtentionContents(file);
```

Comment:
Is it possible to change "var" to "byte[]" here? 

In this case the type is not explicit from the right side of the assignment. When the type is not easily inferred (need to look for the implementation) is better to explicitly refer to it in the local variable definition.

Reply:
Updated